### PR TITLE
Add support for `personal_access_token_request` webhook event type

### PR DESCRIPTION
**Pull request by joaopenteado**
This PR adds support for the [`personal_access_token_request`](https://docs.github.com/en/webhooks-and-events/webhooks/webhook-events-and-payloads#personal_access_token_request) webhook event.

> This event occurs when there is activity relating to a request for a fine-grained personal access token to access resources that belong to a resource owner that requires approval for token access.

### CODE REVIEW

File: github/event_types.go
Comment (gmlewis) [inline github/event_types.go:793]: Please add a godoc-style comment to this new struct.

File: github/event_types.go
Comment (gmlewis) [inline github/event_types.go:836]: Please add a godoc-style comment to this new struct.

**Review by gmlewis — CHANGES_REQUESTED** [review]
Thank you, @joaopenteado !
Just a few tweaks please, then we will be ready for a second LGTM+Approval from any other contributor to this repo before merging.

**Comment by joaopenteado** [comment]
Thanks for the very quick response, @gmlewis!
I've just committed the documentation changes requested. Let me know if you want any additional changes.

**Review by gmlewis — APPROVED** [review]
Thank you, @joaopenteado !
LGTM.

Awaiting second LGTM+Approval from any other contributor to this repo before merging.

**Comment by joaopenteado** [comment]
@gmlewis upon further testing, I've actually found that this webhook delivers an map in the API. This isn't very well documented IMO but could be inferred I guess. I've updated the event type to fix this. Here's an excerpt from a webhook event I tested:

```json
{
  "action": "created",
  "personal_access_token_request": {
    "id": 11233,
    "owner": { /* ... */ },
    "repository_selection": "all",
    "repository_count": null,
    "repositories": null,
    "permissions_added": {
      "organization": {
        "organization_announcement_banners": "write",
        "organization_events": "read"
      },
      "repository": {
        "codespaces_lifecycle_admin": "read",
        "metadata": "read",
        "security_events": "write"
      }
    },
    "permissions_upgraded": {},
    "permissions_result": {
      "organization": {
        "organization_announcement_banners": "write",
        "organization_events": "read"
      },
      "repository": {
        "codespaces_lifecycle_admin": "read",
        "metadata": "read",
        "security_events": "write"
      }
    },
    "created_at": "2023-07-07T00:37:57+09:00",
    "token_expired": false,
    "token_expires_at": "2023-08-05T15:37:56Z",
    "token_last_used_at": null
  },
  "organization": { /* ... */ },
  "sender": { /* ... */},
  "installation": {
    "id": 1111111,
    "node_id": "xxxxxxxxxxxxxx"
  }
```

**Review by gmlewis — APPROVED** [review]
Thank you, @joaopenteado !
LGTM.

Awaiting second LGTM+Approval from any other contributor to this repo before merging.

**Review by liaodaniel — APPROVED** [review]
LGTM

**Comment by gmlewis** [comment]
Thank you, @liaodaniel !
Merging.

**Pull request merged**
